cylc/flow/scheduler_cli.py

```diff
@@ -234,6 +234,12 @@ def get_option_parser(is_restart, add_std_opts=False):
         metavar="PLUGIN_NAME", action="append", dest="main_loop"
     )
 
+    parser.add_option(
+        "--abort-if-any-task-fails", "--fragile",
+        help="If set workflow will abort with status 1 if any task fails.",
```

Comment:
```suggestion
        help="If set scheduler will abort with status 1 if any task fails.",
```

This might be overly pedantic, but I think we should try to distinguish the "workflow" as an abstract structure from the "scheduler" program that manages it.